data/strings/en.ini

```
@@ -90,6 +91,7 @@ job_working = {0}<<Working...||&Cancel
 nothing_to_report = Crash Report<<Nothing to report||&OK
 install_extension = Warning\n<<Do you really want to install the given extension?\n<<"{0}"\n||&Install||&Cancel
 uninstall_extension_warning = Warning\n<<Do you really want to uninstall "{0}" extension?\n||&Yes||&No
+cannot_install_default_extension = Error\n<<This theme cannot be installed because its name matches the default aseprite theme\n||&OK
```

I'd change the message to something like:
```suggestion
cannot_install_default_extension = Error<<This extension cannot be installed because tries to replace the default theme.<<<<Contact the developer so they fix the theme ID string from \"default\" to something else.||&OK
```